### PR TITLE
Updated User UI as new Mockup and also did UC-50 for loan signup

### PR DESCRIPTION
**Pull request by cpatelss90**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -77,23 +84,37 @@ class App extends Component {
...
+              <Link to={"/profile"} className="nav-link">
+                Profile
+              </Link>
+            </li>
           </div>
         </nav>
 
         <div className="container mt-3">
           <Switch>
+            <Route exact path="/home" component={Home} />
```
Comment (Joshua-Podhola) [inline src/App.js:109]: I don't think this is reachable

File: src/components/LoansOnOfferComponent.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+      handleSubmitPersonalLoan() {
+        AuthService.loanSignup(5000.10, 'Personal', 4.10, 1000.34, nowDate.getDate())
+        .then(
+        () => {
+            this.props.history.push("/signupLoan");
+            window.location.reload();                        
+
+        })
+      }
```
Comment (Joshua-Podhola) [inline src/components/LoansOnOfferComponent.js:51]: This ends up registering a new loan type into the database every time a user signs up for one...

File: src/components/SignupLoanComponent.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        super(props);
+    }
+    
+
+    render() {
+        return (
+            <div> You have successfully Signed up for a loan! </div>  
+        )
+
+    }
```
Comment (Joshua-Podhola) [inline src/components/SignupLoanComponent.js:14]: I take it this is a placeholder

File: src/services/authService.js [hunk 4]
```diff
@@ -41,6 +42,19 @@ class AuthService {
...
+    return axios.
+    post(API_URL2 + "loansignup?userId=" + this.getCurrentUser().id
+    , {
+      max_amount,
+      name,
+      interest_rate,
+      balance,
+      start_date
+    })
+  }
```
Comment (Joshua-Podhola) [inline src/services/authService.js:56]: You're passing the interest rate and balance of the loan to the back end...

...nooooooo...

File: src/components/LoansOnOfferComponent.js [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+                        <h7>Mortgage Terms Available: 1, 3, 5, 10 Years</h7>
+                        <h7>Down Payment required: None</h7>
+
+                        <button onClick={this.handleSubmitPersonalLoan}>
+                            Sign Up
+                        </button>
+                    </div>
+                </div>
+
+            </div>
```
Comment (Joshua-Podhola) [inline src/components/LoansOnOfferComponent.js:107]: Wait, so all the loan types are hard coded into the HTML? Like, yeah, it works for demo, but when it comes to pulling the loan types from the database, then almost every single line in this file will need to be thrown out...

**Review by Joshua-Podhola — CHANGES_REQUESTED** [review]
Yes, it fits the mockups just fine, but sure we should be getting the loan information from the database, right...?

File: src/App.js (hunk 1 above)
Comment (cpatelss90) [reply src/App.js:109]: Its reachable, you just have to test it out.

File: src/components/LoansOnOfferComponent.js (hunk 2 above)
Comment (cpatelss90) [reply src/components/LoansOnOfferComponent.js:51]: Again, this is the same issue you brought up in the backend MS, which I explained there...

File: src/services/authService.js (hunk 4 above)
Comment (cpatelss90) [reply src/services/authService.js:56]: Again, this is the same issue you brought up in the backend MS, which I explained there...

File: src/components/LoansOnOfferComponent.js (hunk 5 above)
Comment (cpatelss90) [reply src/components/LoansOnOfferComponent.js:107]: Again, this is the same issue you brought up in the backend MS, which I explained there...

**Review by Joshua-Podhola — APPROVED** [review]

**Pull request merged**